components/formats-bsd/src/loci/formats/tiff/TiffSaver.java

```diff
@@ -419,6 +423,7 @@ private void writeImageIFD(IFD ifd, int no, byte[][] strips,
     int tilesPerColumn = (int) ifd.getTilesPerColumn();
     boolean interleaved = ifd.getPlanarConfiguration() == 1;
     boolean isTiled = ifd.isTiled();
+    long defaultByteCount = 0L;
 
     RandomAccessInputStream in = null;
```

Comment:
other resources do not seem to be closed in ``writeIFD``
This could be fixed in another PR